src/main/java/com/google/sps/data/BusinessesService.java

```diff
@@ -33,10 +40,40 @@
   /** Create a new Businesses instance
   * @param businesses businesses from SmallCityService
   **/
+  
   public BusinessesService(List<Listing> businesses) {
     this.businesses = businesses;
   }
 
+  public PreparedQuery getBigBusinessFromDatabase(){
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("BigBusinesses");
+    PreparedQuery queryOfDatabase = datastore.prepare(query);
+    return queryOfDatabase;
+  }
+
+  public List<Listing> removeBigBusinessesFromResults(PreparedQuery queryOfDatabase){
+    Iterator<Listing> businessesList =  businesses.iterator();
```

Comment:
nit: *businesses 

Try to keep the datatype out of the variable name. One advantage this brings is avoiding having to change the name, if the type changes, or having mismatches.

Reply:
The problem is I already have a variable named businesses as you can see on the right side of the equal sign. So can I leave this variable name as is?

Reply:
Feel free to rename the other variables as necessary. Think of it more like an opportunity to make the code more readable and easier to maintain, than as a problem. 

This is precisely what I'm getting at. If the collection of businesses is not longer a list (say it's changes to a map instead) "businessList" becomes a misnomer, and can add to cognitive overhead understanding what's happening.

Reply:
Okay I see where you are coming from. So I decided to change the name of the main businesses variable in the class, to smallBusinesses as at the end it should only contain small businesses listings. I changed the iterator to just businesses like you recommended I do.